cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -522,18 +520,19 @@ private String getClassPath(JBallerinaBackend jBallerinaBackend, Package current
 
         for (ModuleId moduleId : currentPackage.moduleIds()) {
             Module module = currentPackage.module(moduleId);
-
-            // Skip adding the path if the module doesnt contain a generated module jar
-            if (!module.documentIds().isEmpty()) {
+            try {
                 PlatformLibrary generatedJarLibrary = jBallerinaBackend.codeGeneratedLibrary(
                         currentPackage.packageId(), module.moduleName());
                 exclusionPathList.add(generatedJarLibrary.path());
+            } catch (IllegalStateException e) {
```

Comment:
Why do we have to catch the `IllegalStateException` here?

Reply:
In retrospect, since the source jar is created regardless of whether there is a source file or not, I suppose the exception will not be thrown and can be safely added to the list

Reply:
I have removed the try/catch block